doc/OnlineDocs/explanation/analysis/alternative_solutions.rst

```diff
@@ -2,233 +2,359 @@
+               "name": "o",
+               "suffix": {},
+               "value": 70.0
+           }
+       ],
+       "suffix": {},
+       "variables": [
+           {
+               "discrete": true,
+               "fixed": false,
+               "index": 0,
+               "name": "x[0]",
+               "suffix": {},
+               "value": 0
+           },
+           {
+               "discrete": true,
+               "fixed": false,
+               "index": 1,
+               "name": "x[1]",
+               "suffix": {},
+               "value": 1
+           },
+           {
+               "discrete": true,
+               "fixed": false,
+               "index": 2,
+               "name": "x[2]",
+               "suffix": {},
+               "value": 1
+           },
+           {
+               "discrete": true,
+               "fixed": false,
+               "index": 3,
+               "name": "x[3]",
+               "suffix": {},
+               "value": 0
+           }
+       ]
    }
    {
-       "fixed_variables": [],
-       "objective": "o",
-       "objective_value": 12.0,
-       "solution": {
-           "x[0]": 0,
-           "x[1]": 1,
-           "x[2]": 1,
-           "x[3]": 1,
-           "x[4]": 0
-       }
-   }
-   {
-       "fixed_variables": [],
-       "objective": "o",
-       "objective_value": 12.0,
-       "solution": {
-           "x[0]": 1,
-           "x[1]": 0,
-           "x[2]": 0,
-           "x[3]": 1,
-           "x[4]": 1
-       }
-   }
-   {
-       "fixed_variables": [],
-       "objective": "o",
-       "objective_value": 12.0,
-       "solution": {
-           "x[0]": 1,
-           "x[1]": 0,
-           "x[2]": 1,
-           "x[3]": 0,
-           "x[4]": 0
-       }
+       "id": 0,
+       "objectives": [
+           {
+               "index": 0,
+               "name": "o",
+               "suffix": {},
+               "value": 70.0
+           }
+       ],
+       "suffix": {},
+       "variables": [
+           {
+               "discrete": true,
+               "fixed": false,
+               "index": 0,
+               "name": "x[0]",
+               "suffix": {},
+               "value": 1
+           },
+           {
+               "discrete": true,
+               "fixed": false,
+               "index": 1,
+               "name": "x[1]",
+               "suffix": {},
+               "value": 0
+           },
+           {
+               "discrete": true,
+               "fixed": false,
+               "index": 2,
+               "name": "x[2]",
+               "suffix": {},
+               "value": 0
+           },
+           {
+               "discrete": true,
+               "fixed": false,
+               "index": 3,
+               "name": "x[3]",
+               "suffix": {},
+               "value": 1
+           }
+       ]
    }
 
+Further, variable and objective values can be retrieved either using an integer index or the corresponding name:
```

Comment:
Would it be hard to allow retrieving them using the Var or Objective object itself?

Reply:
So the solution pool data would depend on links to an active model?  What if I loaded a solution pool from a file?

Reply:
I am going to leave this functionality to a follow-on PR.  The design of solution pools intentionally omits references to model component data.